app/views/shared/_feed.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if @feed_items.any? %>
+  <ol class="microposts">
+    <%= render @feed_items %>
```

Comment:
cái này hoạt động ra sao

Reply:
có thể in ra một danh sách feed thay vì dùng lệnh each để in ra